PocketCastsTests/Tests/Player/Transcripts/TranscriptModelTests.swift

```diff
@@ -0,0 +1,124 @@
+import XCTest
+
+@testable import podcasts
+
+final class TrasnscriptModelFilterTests: XCTestCase {
```

Comment:
typo

```suggestion
final class TranscriptModelFilterTests: XCTestCase {
```

Reply:
Fixed on: [Rename file and test](https://github.com/Automattic/pocket-casts-ios/pull/1870/commits/99ca08946ea505c2379e447af7f7335a29d1bdc1)